data/array/type tester fork/19aa85fb-32fd-4877-a6c8-96aa1865e89e.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "data/array/Type tester fork",
+  "interface": {
+    "inputs": {
+      "498fba4a-b825-49aa-a999-b46f46c9bc46": {
+        "type": "any",
+        "name": "data",
+        "description": "Receives the data to be forwarded."
+      }
+    },
+    "outputs": {
+      "54739c09-5a28-4237-9e38-38e8aed6b189": {
+        "type": "any[]",
+        "name": "on array",
```

Comment:
Other typed fork keep the output ports named as `on true` and `on false`. For consistency we could use this same naming scheme here too.